packages/elements/src/react/hooks/use-loading.hook.ts

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable react-hooks/rules-of-hooks */
+import { useActiveTags } from '~/react/hooks/use-active-tags.hook';
+import { SignInStartCtx } from '~/react/sign-in/start';
+import { SignInFirstFactorCtx, SignInSecondFactorCtx } from '~/react/sign-in/verifications';
+
+/**
+ * Caution: This hook is unstable and may disappear in the future.
+ * This is a temporary hook until the actual loading API is explored and implemented.
+ */
+export const unstable_useIsLoading = () => {
```

Comment:
To keep with React hook naming conventions:


```suggestion
export const useIsLoading_unstable = () => {
```